app/(datasets)/data-catalog/page.tsx

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
+import dynamic from 'next/dynamic';
 import { getTransformedDatasetMetadata } from 'app/content/utils/mdx';
 import { Suspense } from 'react';
-import Catalog from './catalog';
+
+const Catalog = dynamic(
+  () => import('./catalog'),
+  { 
+    ssr: false,
+    loading: () => <p>Loading...</p> // @NOTE @TODO: We need a loading state!!!
```

Comment:
The dynamic import is not necessary since we are using <Suspense>. The <Suspense> component can handle the loading state for us.